### ISSUE TITLE
Feature request - Capture button mapping

### ISSUE THREAD

**Issue by AidenIkeda** [Ryujinx/Ryujinx#3875]
Many games require the capture button to perform tasks like the photo mode in Pokemon Scarlet and Violet. I believe Yuzu has this feature so I don't see any reason for this not to be implemented.

**Comment by TSRBerry** [comment]
>  I believe Yuzu has this feature so I don't see any reason for this not to be implemented.

I gotta say this is pretty bad reasoning. Nothing against the feature request itself, but I don't understand why so many people need to justify everything with Yuzu.

**Comment by gdkchan** [comment]
I don't think games make any actual use of this button (I think they can't even access it). So ultimately it doesn't sounds like it would be a very useful feature. The emulator already has a hotkey to take screenshots. Maybe we should allow that and other hotkeys to be mapped to controllers which seems more useful.

**Comment by EmulationFanatic** [comment]
Bayonetta 3 uses the capture button in Photo mode. You could technically take screenshots there instead, but to say no games make use of the button isn't entirely accurate.

**Comment by gdkchan** [comment]
I meant using it for something other than taking a screenshot (which can already be done using several other methods in a emulator).

**Comment by Mimoulem** [comment]
A game of a champion, the electic need to take screenshot to go for the battle in Scarlet. If we haven't this screen shot button, we can't continue the game. Please, map the screenshot button for switch we are all stuck.

**Comment by gdkchan** [comment]
That button is not required to complete the game. Several people already completed the game on emulator. Please show a screenshot of the place where the game requests the use of this button.

**Comment by Mimoulem** [comment]
![image](https://user-images.githubusercontent.com/118673138/202907344-7977fc17-7358-40f5-a0ba-02f8ce96cc35.png)
I find the guy for the game, i spam the A button  but nothing happen, i tried all of them, except the home and screenshot button that ar non mapped on Ryujinx. Do you have some answers ?

**Comment by gdkchan** [comment]
Some people said that increasing the controller deadzone in the settings fixes the issue, you can try that.

**Comment by Mimoulem** [comment]
Oh, it works. Incredible thanks a lot, you're a beautiful person ! Thanks thanks thanks thanks thanks !

**Comment by AidenIkeda** [comment]
Sure, it's not that useful. But there's really no reason for it not to be implemented. Maybe future games would utilize more of the capture button, and I'm sure some games uses it as well. Other emulators do it so why not Ryujinx? It's a basic feature.

**Comment by gdkchan** [comment]
If games can actually access the state of those buttons then I agree that we should implement it. But I think games can't since it has special functionality just like the home button, that is handled by the firmware. If it's the latter I don't think there's much point in implementing it on the emulator unless we plan to do LLE of more firmware services in the future.

**Comment by Legarad** [comment]
In Pokemon Scarlet and Violet the capture button is used to customize your student profile.

**Comment by neiviv-dev** [comment]
It's quite a little thing but there is a moon that you can't get in Mario Odyssey without using a screenshot

**Comment by andrewmyles2** [comment]
> Oh, it works. Incredible thanks a lot, you're a beautiful person ! Thanks thanks thanks thanks thanks !

yeah,. but it's a workaround. there should JUST be abilityto map buttons that exist on joycons,. Imagine the devs forgot about X button. Or left trigger.

**Comment by riperiperi** [comment]
It's not a workaround, it's your deadzone being set too low for your controller. The game doesn't want you selecting a target while still moving the cursor. The SMO map screen has a similar behaviour, except unsurprisingly you can't press the capture button for that one.

**Comment by Psyda** [comment]
Why is this still not resolved?

It's needed in Botw to photograph creatures and Tears of the Kingdom.

**Comment by AcK77** [comment]
> That button is not required to complete the game. [...] Please show a screenshot of the place where the game requests the use of this button.

As stated above!

**Comment by Psyda** [comment]
> > That button is not required to complete the game. [...] Please show a screenshot of the place where the game requests the use of this button.
> 
> As stated above!

What a pointless thing to debate. It cuts a chunk of the games mechanics out in Botw and Totk. I can't even zoom in as it uses the camera button.

**Comment by AcK77** [comment]
The capture button just do screenshots which are stored in the Switch Album savedata. This is not used at all in games. The camera feature of the Pru'ha tablet in Totk is anything else and not related at all with the capture button.

**Comment by MutantAura** [comment]
> What a pointless thing to debate. It cuts a chunk of the games mechanics out in Botw and Totk. I can't even zoom in as it uses the camera button.

It is pointless to lie. The zoom function is mapped to RStick press-in, not the screenshot button. 
The camera in BoTW and ToTK is activated by Rune Slate functions, either via holding Dpad Up in BoTW or L in ToTK. I would recommend you check the controls of the game you're playing before commenting fairly aggressive statements.

**Comment by Psyda** [comment]
> > What a pointless thing to debate. It cuts a chunk of the games mechanics out in Botw and Totk. I can't even zoom in as it uses the camera button.
> 
> It is pointless to lie. The zoom function is mapped to RStick press-in, not the screenshot button. The camera in BoTW and ToTK is activated by Rune Slate functions, either via holding Dpad Up in BoTW or L in ToTK. I would recommend you check the controls of the game you're playing before commenting fairly aggressive statements.

That was not my intention. I vaguely remember using capture to store the creatures. Apologies if I misremembered. I used to use a different emulator. Coming back for Totk I'm noticing I can't fill up the album. It just doesn't have a key to take pictures any more. Looking it up I was told to use the Capture button.

**Comment by juliocargnin** [comment]
I can't do some moon missions without the capture button in Mario Odyssey! 😢😢😢
